### PR TITLE
Try and make the formatting and English a little more consistent

### PR DESCRIPTION
**Pull request by pjlsergeant**
* Wrap specifically at 80 rather than a more random approach
* Little English language fixups, including making it more

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -32,28 +32,34 @@ have prepared these guidelines for you:
...
-Verdaccio has several areas of involvement, which might fit you better, eg:
+The Verdaccio project is split into several areas:
 
-- **Core**: The [core](https://github.com/verdaccio/verdaccio) is the main repository, built with **Node.js**.
-- **Website**: we uses **Gatsby.js** for the **website** and if you are familiar with this technology, you might become the official webmaster.
-- **User Interface**: The [user Interface](https://github.com/verdaccio/ui) is based in **react** and **material-ui** and looking for front-end contributors.
-- **Kubernetes and Helm**: Ts the official repository for the [**Helm chart**](https://github.com/verdaccio/charts).
+- **Core**: [core](https://github.com/verdaccio/verdaccio) is the main
+repository, built with **Node.js**
+- **Website**: we use **Gatsby.js** for the **website** and if you are familiar
```
Comment (juanpicado) [inline CONTRIBUTING.md:48]: Please,  replace `Gatsby.js` by `Docusaurus`, we decided not to continue with `Gatsby.js` for some reasons #1862

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -2,8 +2,8 @@
 
 > Any change matters, whatever the size, just do it.
 
-We are happy that you are willing to contribute. For that reason, we
-have prepared these guidelines for you:
+We're happy that you're considering contributing! To help, we've prepared these
```
Comment (juanpicado) [inline CONTRIBUTING.md:5]: I'd prefer don't use abbreviation on the documentation, looks a bit informal. Could we restore them?

https://www.grammarly.com/blog/abbreviations/

> One thing to remember about abbreviations is that certain ones are considered informal. If you are writing something very formal, it’s better to err on the side of spelling things out. The other thing to remember is that some readers may not know what an abbreviation means. If the abbreviation is obscure or unfamiliar, make sure to explain what it means the first time you use it.

Example: https://yarnpkg.com/advanced/contributing

Comment (pjlsergeant) [reply CONTRIBUTING.md:5]: @juanpicado You're quoting text that suggests this rule applies when "writing something very formal" and you've linked as an example to a piece of text that uses a very similar abbreviation 3 sentences in...?

Comment (juanpicado) [reply CONTRIBUTING.md:5]: Yeah, I bad example from my side. My point is to be consistent with the current documentation, eg: https://github.com/verdaccio/website/blob/master/docs/best-practices.md

Eventually those files will be hosted also in this repo. I'm not a technical writer, so I'm not giving advice in such level, maybe you are I don't know. 

I was just suggesting consistency. Somehow I appreciate you improve the reading 👍🏼

**Pull request merged**